docs/docs/libstart.md

```diff
@@ -14,7 +14,7 @@ sidebar_label: zos-lib quickstart
 To install `zos-lib` simply go to your project's root directory and run:
 
 ```sh
-npm i zos-lib
+npm install zos-lib
```

Comment:
I agree with this edit.

Note: Should the _--save_ flag be used here to save the pakcage to the _package.json_ file for the project?